src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PostCodeDistrictEntity.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import java.math.BigDecimal
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "postcode_districts")
+data class PostCodeDistrictEntity(
+  @Id
+  val id: UUID,
+  val outcode: String,
+  val latitude: BigDecimal,
+  val longitude: BigDecimal
```

Comment:
In our experiments in alpha we used the `geometry(point, 4326)` type for PostGIS. Are you thinking of adding that later once we get into querying the data? I do think it would probably make sense to store the coordinates separately as lat and lng also, at least initially...

Reply:
I hadn't thought that far ahead if I'm honest but I think I'll merge this in as-is then look at PostGIS as will need to do some work to install it if we go that way